jupyter_notebooks/pylibCZIrw/pylibCZIrw_3_3_0.ipynb

```diff
@@ -1279,7 +1280,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.15"
+   "version": "3.9.16"
+  },
+  "vscode": {
+   "interpreter": {
```

Comment:
Change not needed.
```suggestion
```

Reply:
This also does not matter in my opinion, because every user will change this